Sources/NIOPosix/Bootstrap.swift

```diff
@@ -474,17 +479,17 @@ public final class ServerBootstrap {
             func fireThroughPipeline(_ future: EventLoopFuture<Void>, context: ChannelHandlerContext) {
                 ctxEventLoop.assertInEventLoop()
                 assert(ctxEventLoop === context.eventLoop)
-                let loopBoundContext = context.loopBound
+                let loopBoundValues = NIOLoopBound((context: context, self: self), eventLoop: context.eventLoop)
```

Comment:
Can't we use the assume isolated APIs here?

Reply:
Yes, we can. Updated.

Reply:
Love it!